code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -858,11 +858,10 @@
 	switch(action)
 		if("drying")
 			drying = !drying
-			change_power_mode(drying ? ACTIVE_POWER_USE : IDLE_POWER_USE)
 			update_icon(UPDATE_OVERLAYS)
 
 /obj/machinery/smartfridge/drying_rack/update_overlays()
-	if(stat & NOPOWER)
+	if(stat)
 		. += "drying_rack_off"
```

Comment:
Shouldn't you just delete these lines and remove the no power sprite?

Reply:
I tested the original and my drying rack procedures. I didn't see a sprite difference

Reply:
There is a sprite difference, go check out the .dmi file!

Reply:
I tested it again, and yeah everything is consistent.

Reply:
Just because there is no difference in-game doesn't mean that there isn't a sprite for it.

Reply:
I don't see an inconsistency in the functioning sprite in game.

Reply:
It. Is. Not. In-game.
Check the .dmi file.

Reply:
wait a minute, do you want me to delete the no power sprite?

Reply:
Yes
If you aren't gonna use the sprite, you should delete it
Also, `stat & NOPOWER` means that it checks if the stat has no power, meaning that the sprite will always have the no-power sprite.

Reply:
Peculiar, the no-power sprite was actually never present during my testing. It ran the base sprite and the drying sprite, so I suppose removing the lines is just cleaning up.